src/cljs/replumb/repl.cljs

```diff
@@ -777,12 +817,14 @@
   It accepts a sequence of symbols or strings."
   ([]
    (reset-env! nil))
-  ([namespaces]
-   (read-eval-call {} identity "(in-ns 'cljs.user)")
+  ([opts]
+   (reset-env! opts nil))
+  ([opts namespaces]
```

Comment:
A note on this. If we don't pass the same options as in the test to the `read-eval-call` below, the system will think that we want to re-initialize and therefore wreaking havoc. That is why all the `reset-env!` calls have changed.